service/pom.xml

```diff
@@ -22,14 +22,15 @@
         <tds.session.client>0.0.1-SNAPSHOT</tds.session.client>
         <tds.common>0.0.1-SNAPSHOT</tds.common>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
         <unit-tests.skip>false</unit-tests.skip>
         <integration-tests.skip>true</integration-tests.skip>
         <docker.image.prefix>sbacoss</docker.image.prefix>
         <assertj.version>3.4.1</assertj.version>
         <mockito.version>1.10.19</mockito.version>
         <rest-assured.version>2.9.0</rest-assured.version>
         <h2-database.version>1.4.191</h2-database.version>
+        <java.version>1.8</java.version>
+        <maven.compiler.plugin.version>3.5.1</maven.compiler.plugin.version>
```

Comment:
Yes, this happens because SpringBoot is the parent to this project.  I suggest making SpringBoot the parent of the pom at the base of the repository.  I'd then make the parent of this and client the pom at the parent.  It may be easier we just chat.